### PR TITLE
Fixed RViz default plugin license linter

### PR DESCRIPTION
**Pull request by ahcorde**

### CODE REVIEW

File: rviz_default_plugins/include/rviz_default_plugins/displays/accel/accel_display.hpp [hunk 1]
```diff
@@ -1,32 +1,30 @@
...
- * SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
- * INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
- * CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
- * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
- * POSSIBILITY OF SUCH DAMAGE.
- */
-
+// Copyright (c) 2023, Open Source Robotics Foundation, Inc.
+// All rights reserved.
+// Redistribution and use in source and binary forms, with or without
```
Comment (clalancette) [inline rviz_default_plugins/include/rviz_default_plugins/displays/accel/accel_display.hpp:4]: Nit, but here and in a bunch of other places I think a blank line is in order (it is in some of the copyright headers, but not all; we should be consistent):

```suggestion

// Redistribution and use in source and binary forms, with or without
```

**Review by clalancette — CHANGES_REQUESTED** [review]
I didn't check all of them, but this is a good cleanup.  I'd like to see us be consistent with the space between the "All rights reserved." and "Redistribution..." line.

File: rviz_default_plugins/include/rviz_default_plugins/displays/pointcloud/point_cloud_common.hpp [hunk 2]
```diff
@@ -1,31 +1,30 @@
...
- * ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE
- * LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
- * CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
- * SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
- * INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
- * CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
- * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
- * POSSIBILITY OF SUCH DAMAGE.
- */
+// Copyright (c) 2012, Willow Garage, Inc.
```
Comment (clalancette) [inline rviz_default_plugins/include/rviz_default_plugins/displays/pointcloud/point_cloud_common.hpp:1]:
```suggestion
// Copyright (c) 2012, Willow Garage, Inc.
// All rights reserved.

```

**Review by clalancette — CHANGES_REQUESTED** [review]
One more minor fix, then this is ready for CI.

**Comment by ahcorde** [comment]
* Linux [Build Status](http://ci.ros2.org/job/ci_linux/21448/)
* Linux-aarch64 [Build Status](http://ci.ros2.org/job/ci_linux-aarch64/15736/)
* Linux-rhel [Build Status](http://ci.ros2.org/job/ci_linux-rhel/950/)
* Windows [Build Status](http://ci.ros2.org/job/ci_windows/22168/)

**Review by clalancette — APPROVED** [review]

**Pull request merged**